dotCMS/src/main/java/com/dotmarketing/quartz/job/CleanUpFieldReferencesJob.java

```diff
@@ -52,12 +48,28 @@
 
     @Override
     @WrapInTransaction
-    public void run(JobExecutionContext jobContext) throws JobExecutionException {
+    public void run(final JobExecutionContext jobContext) throws JobExecutionException {
 
-        final User user = (User)jobContext.getJobDetail().getJobDataMap().get("user");
-        final Field field = (Field) jobContext.getJobDetail().getJobDataMap().get("field");
-        final Date deletionDate = (Date) jobContext.getJobDetail().getJobDataMap().get("deletionDate");
+        final Trigger trigger = jobContext.getTrigger();
+        final Optional<Map<String, Object>> triggerJobDetailOptional = getTriggerJobDetail(CleanUpFieldReferencesJob.class);
+        if(!triggerJobDetailOptional.isPresent()){
+            throw new IllegalArgumentException(
+                    String.format("Unable to get job detail data %s ", trigger.getName()));
+        }
+            final Map<String, Object> triggerJobDetail = triggerJobDetailOptional.get();
+            @SuppressWarnings("unchecked")
+            final Map<String, Serializable> executionData = (Map<String, Serializable>) triggerJobDetail.get(trigger.getName());
+            if(null == executionData) {
+                throw new IllegalArgumentException(
+                        String.format("Unable to get trigger execution data for trigger `%s` ", trigger.getName()));
+            }
+
+        final User user = (User)executionData.get("user");
+        final Field field = (Field) executionData.get("field");
+        final Date deletionDate = (Date) executionData.get("deletionDate");
 
+        System.out.println(":::Job-Started::Thread:" + Thread.currentThread().getName() +"::"+field.name());
```

Comment:
Codacy found an issue: [System.out.println is used](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6077927)

Reply:
remove this???